.script/tests/detectionTemplateSchemaValidation/DetectionTemplateSchemaValidationTests.cs

```diff
@@ -107,18 +108,21 @@ public void Validate_DetectionTemplates_TemplatesThatAreInTheWhiteListShouldNotP
         [Fact]
         public void Validate_DetectionTemplates_AllFilesAreYamls()
         {
-            string detectionPath = DetectionsYamlFilesTestData.GetDetectionPath();
-            var yamlFiles = Directory.GetFiles(detectionPath, "*.yaml", SearchOption.AllDirectories).ToList();
-            var AllFiles = Directory.GetFiles(detectionPath,"*", SearchOption.AllDirectories).ToList();
-            var numberOfNotYamlFiles = 1; //This is the readme.md file in the directory
-            Assert.True(AllFiles.Count == yamlFiles.Count + numberOfNotYamlFiles,  "All the files in detections folder are supposed to end with .yaml");
+            List<string> detectionPath = DetectionsYamlFilesTestData.GetDetectionPath();
+            var yamlFiles = Directory.GetFiles(detectionPath[0], "*.yaml", SearchOption.AllDirectories).ToList(); // Detection folder
+            yamlFiles.AddRange(Directory.GetFiles(detectionPath[1], "*.yaml", SearchOption.AllDirectories).ToList().Where(s=>s.Contains("Analytic Rules"))); // Extending detection validation to solution folder
+            var AllFiles = Directory.GetFiles(detectionPath[0],"*", SearchOption.AllDirectories).ToList();
+            AllFiles.AddRange(Directory.GetFiles(detectionPath[1], "*", SearchOption.AllDirectories).ToList().Where(s => s.Contains("Analytic Rules")));
+            var numberOfNotYamlFiles = 2; //This is the readme.md file in the detection and Cisco Ise soluton directory.
```

Comment:
@v-maudan, Please remove empty readme file from CIsco ISE of solutions analytics folder and make it 1 (as earlier).

Reply:
done.